### PR TITLE
Bento-39 Program Listing config

### PR DESCRIPTION
**Pull request by skiran86**
## Description

Bento-39 Program Listing config

### CODE REVIEW

File: src/pages/programs/programsView.js [hunk 1]
```diff
@@ -33,116 +55,33 @@ const Programs = ({ classes, data }) => {
...
+              <span className={classes.linkSpan}>
+                <a href={`${column.externalLink.replace('{}', value)}`} target="_blank" rel="noopener noreferrer" className={classes.link}>{value}</a>
+                <img
+                  src={externalLinkIcon.src}
+                  alt={externalLinkIcon.alt}
+                  className={classes.externalLinkIcon}
+                />
+              </span>
+            )
+              : column.field === 'num_subjects' ? <Link className={classes.link} to={(location) => ({ ...location, pathname: '/cases' })} onClick={() => redirectTo('TAILORx')}>{value}</Link>
```
Comment (AjayDoddapaneni) [inline src/pages/programs/programsView.js:78]: Is TAILORx hardcoded value breaks when we add a new Program?

File: src/bento/programData.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      label: 'Start Date',
+    },
+    {
+      field: 'end_date',
+      label: 'End Date',
+    },
+    {
+      field: 'pubmed_id',
+      label: 'PubMed ID',
+      externalLink: 'https://pubmed.ncbi.nlm.nih.gov/{}',
```
Comment (AjayDoddapaneni) [inline src/bento/programData.js:41]: If you use this check you can keep just link for both internal and external links
function linkIsExternal(linkElement) {
  return (linkElement.substring(0, 1) !== '/');
}

File: src/pages/programs/programsView.js [hunk 3]
```diff
@@ -1,21 +1,48 @@
...
+  table, tableTitle, icon, externalLinkIcon,
+} from '../../bento/programData';
 import Stats from '../../components/Stats/AllStatsController';
 import { Typography } from '../../components/Wrappers/Wrappers';
-import icon from '../../assets/trial/Trials_Title_Bar.Icon.svg';
-import externalLinkIcon from '../../assets/icons/Program-ExternalLink.svg';
 import { singleCheckBox, fetchDataForDashboardDataTable } from '../dashboard/dashboardState';
 
+function manipultateLinks(tableData) {
+  let cloumnCount = 0;
```
Comment (AjayDoddapaneni) [inline src/pages/programs/programsView.js:22]: You can remove this

File: src/pages/programs/programsView.js [hunk 4]
```diff
@@ -1,21 +1,48 @@
...
+  tableData.forEach((column) => {
```
Comment (AjayDoddapaneni) [inline src/pages/programs/programsView.js:23]: replace this with ableData.forEach((column, index) => {
index will go ve the columCount

Comment (skiran86) [reply src/pages/programs/programsView.js]: done

**Pull request merged**